libs/packages/layouts/src/lib/feature/search-list-layout/search-list-layout.component.ts

```diff
@@ -179,11 +187,40 @@ export class SearchListLayoutComponent implements OnChanges, OnInit {
     this.filterData = filter;
     this.page.pageNumber = this.page.default ? this.page.pageNumber : 1;
     this.page.default = filter ? false : true;
-    this.updateContent();
+    this.isDefaultFilter(filter);
+    if (!this.isDefaultModel) {
+      this.updateContent();
```

Comment:
I think this will also prevent navigation changes from happening when users go back to default model - updateContent does both navigation and api call, we want to only block the api call, but allow the navigation for history tracking